src/olympia/lib/settings_base.py

```diff
@@ -271,28 +271,26 @@ def get_db_config(environ_var, atomic_requests=True):
     'SECRET_KEY',
     default='this-is-a-dummy-key-and-its-overridden-for-prod-servers')
 
-# Templates
+# Templates configuration.
+# List of path patterns for which we should be using Django Template Language.
 JINJA_EXCLUDE_TEMPLATE_PATHS = (
+    # All emails should be processed with Django for consistency.
+    r'^.*\/emails\/',
+
+    # ^admin\/ covers most django admin templates, since their path should
+    # follow /admin/<app>/<model>/*
     r'^admin\/',
```

Comment:
if this is django's admin templates shouldn't this be in the "Third-party apps + django." section below?

Reply:
It's our own admin templates as opposed to third-party.

Reply:
right, it's matching `admin` folder within `templates` for each app...  In that case, follow-up of moving `blocklist` and `addons` admin templates into the correct places so we don't have to special case them?

Reply:
Yeah I was too lazy to do this after having spent the day on various extraction-related issues, but you're right it'd be cleaner. I filed https://github.com/mozilla/addons-server/issues/14053